cosmotech/data_update_quest/core/redis/dump_data.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (C) - 2025 - Cosmo Tech
+# This document and all information contained herein is the exclusive property -
+# including all intellectual property rights pertaining thereto - of Cosmo Tech.
+# Any use, reproduction, translation, broadcasting, transmission, distribution,
+# etc., to any person is prohibited unless it has been previously and
+# specifically authorized by written means by Cosmo Tech.
+
+import redis
+import json
+from pathlib import Path
+
+from cosmotech.orchestrator.utils.translate import T
+
+from cosmotech.data_update_quest_cli.utils.logger import LOGGER
+
+
+def configuration(host, port, password):
+    LOGGER.info(T("data_update_quest.core.redis_dump.redis_connection"))
+    r = redis.Redis(host=host, port=port, password=password, decode_responses=True)
+
+    domains = {}
+    domain_list = r.execute_command("FT._LIST")
+    LOGGER.info(T("data_update_quest.core.redis_dump.redis_index"))
+    for domain in domain_list:
```

Comment:
Open question :
What if we wanted to filter the kind of domains we want to dump ?
(for example the 3.3 scenarios/scenarioruns only because we only want to work on those)